Tests/Functional/ControllerResolverTest.php

```diff
@@ -67,4 +69,19 @@ public function testAnnotationControllerServiceExtendingClassicService()
 
         $this->assertEquals('hello', $client->getResponse()->getContent());
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testInvokableControllerAsService()
+    {
+        if (version_compare(Kernel::VERSION_ID, '20600') < 0) {
```

Comment:
@Ener-Getick I checked why i added it in first place (because it was crashing tests).

With `--prefer-lowest` composer flag it is using version `~2.0` of `sensio/framework-extra-bundle`
https://github.com/schmittjoh/JMSDiExtraBundle/blob/master/composer.json#L33

And invokable controllers are not supported in `ParamConverterListener` listener until `3.0.3` https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/v3.0.3/EventListener/ParamConverterListener.php#L66-L67

Symfony framerowk is using `^3.0.2` of  `sensio/framework-extra-bundle` https://github.com/symfony/symfony-standard/blob/2.3/composer.json#L19

So version of `sensio/framework-extra-bundle` should be same here as well in order this test to pass without symfony version restriction even with `--prefer-lowest` flag.